neurovault/urls.py

```diff
@@ -409,6 +419,21 @@ def images(self, request, pk=None):
             page, context={'request': request}, many=True)
         return paginator.get_paginated_response(serializer.data)
 
+    def add_image(self, request, pk=None):
+        collection = get_collection(pk, request, mode='api')
+
+        if not owner_or_contrib(request, collection):
+            self.permission_denied(request)
+
+        image = StatisticMap(collection=collection)
+        serializer = EditableStatisticMapSerializer(data=request.data,
+                                                    instance=image)
```

Comment:
Since NIDM results is not an image_type per se I would prefer going with three different endpoints:

* http://neurovault.org/api/images/ (for statmaps)
* http://neurovault.org/api/atlases/ (for atlases)
* http://neurovault.org/api/nidm_results/ (for NIDM results)